src/entity/User.ts

```diff
@@ -0,0 +1,29 @@
+import {
+    Entity,
+    PrimaryGeneratedColumn,
+    Column,
+    Unique,
+  } from 'typeorm';
+  
+  @Entity()
+  @Unique(['email'])
+  export class User {
+    @PrimaryGeneratedColumn()
+    id!: number;
+  
+    @Column()
+    firstName!: string;
+  
+    @Column()
+    lastName!: string;
+  
+    @Column()
+    email!: string;
+  
+    @Column()
+    password!: string;
+  
+    @Column({ default: "User"} )
+    userType!: string;
+  }
+  
```

Comment:
@maxCastro1 Besides what we have, think about adding more info(fields) about users, such as accountStatus (to indicate whether it is active or inactive/blocked ), contact info (phone number, physical adrress),  etc...

Reply:
@maxCastro1 
Please consider these feedbacks !

Reply:
yes i have added ,  the fields check and see if there any i missed